pyartcd/pyartcd/jenkins.py

```diff
@@ -56,10 +56,13 @@ async def trigger_rhcos(build_version: str, new_build: bool):
     )
 
 
-async def trigger_build_sync(build_version: str):
+async def trigger_build_sync(build_version: str, params: dict):
```

Comment:
Not sure I like this approach. Function prototype does not make evident what params we should expect. Besides, `build_version` gets unfilled when `params` are provided (see the warning in Pycharm [here](https://github.com/openshift-eng/aos-cd-jobs/pull/3605/files#diff-6aeaac93bd7be540b5ddf9fe24d84ee825ae3d9d5740ce4b79f2a2b320af1292R216)). I'd rather do something like:
```
async def trigger_build_sync(build_version: str, doozer_data_gitref: str='', triggered_from_gen_assembly: bool=False):
    
    await trigger_jenkins_job(
        job_path='job/triggered-builds/job/build-sync',
        params={
            'BUILD_VERSION': build_version},
            'DOOZER_DATA_GITREF': doozer_data_gitref,
            'TRIGGERED_FROM_GEN_ASSEMBLY': triggered_from_gen_assembly
        }
    )
```

Reply:
Understood, making the change

Reply:
Fixed in https://github.com/openshift-eng/aos-cd-jobs/pull/3605/commits/43560fc58fd4e127fa655bd82dda758d8965b529